packages/react-cli/src/commands/generate.ts

```diff
@@ -42,6 +46,25 @@ export class GenerateCommand extends Command {
       generatorArgs.push(`--path=${flags.route}/-components`);
     }
 
-    await runEmberInteractively(generatorArgs.join(' '));
+    if(args.generator === 'material'){
```

Comment:
this is the same problem as before, where we have addon-specific things in the generator command.

I guess I was just asking if you thought it'd make sence to make a _Command_ for material. 

like, `export class GenerateMaterialComponent extends Command {...}`

maybe it'd be invoked via:
```
react material path/to/file
# then interactive stuff
```
:-\

Reply:
Okay I can see that, from the original comment, I was only thinking about this in the scope of the blueprints. I think breaking it out to a command does make sense when working with a library like this.